### PR TITLE
refactor: onboarding steps

### PR DESCRIPTION
**Pull request by eivenlour**
**Changes**
- Refactored some of the onboarding stuff so that only the center step content slides in and out of the screen instead of navigating to a whole new screen each step.

**UI** (Updated gifs)

On iPhone 11 Pro Max:

> 

On iPhone SE:

>

### CODE REVIEW

File: lib/common/circle_indicators.dart [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+
+class CircleIndicators extends StatelessWidget {
```
Comment (HelenGezahegn) [inline lib/common/circle_indicators.dart:3]: OMG, I love you ❤️

File: lib/ftu/onboarding/steps.dart [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+  void getSkipPage() {
+    Navigator.pushReplacement(
+      context,
+      MaterialPageRoute(builder: (context) => LoginCreateAccount())
+    );
+  }
+
+  Widget buildSlidingContent() {
+    return SlideInRight(
```
Comment (HelenGezahegn) [inline lib/ftu/onboarding/steps.dart:70]: How does it look when we use the `FadeInRight`?

File: lib/constants/ftu_constants.dart [hunk 3]
```diff
@@ -20,29 +19,45 @@ const String onboardingPartThreeText =
...
+GlobalKey<AnimatorWidgetState> onboardingTextPartThreeKey
+  = GlobalKey<AnimatorWidgetState>();
+
+// Onboarding Steps Config
+List<Map<String, dynamic>> onboardingStepsConfig = [
   {
     "text": onboardingPartOneText,
-    "image": Image.asset('images/onboarding/ano_standing.png', height: imageHeight),
-    "nextScreen": (context) => OnboardingScreens(screenNum: 1), 
+    "image": 'images/onboarding/ano_standing.png',
```
Comment (eivenlour) [inline lib/constants/ftu_constants.dart:40]: I decided to just pass the image path so we can set the image width depending on the screen width as well!

File: lib/ftu/onboarding/steps.dart (hunk 2 above)
Comment (eivenlour) [reply lib/ftu/onboarding/steps.dart]: Here is what `FadeInRight` looks like:



I think `SlideInRight` looks better? What do you guys think? @HelenGezahegn @xiangxinyue @nayancprakash

**Pull request merged**